fakeredis.py

```diff
@@ -331,12 +331,11 @@ def release(self):
         self.redis.delete(self.name)
 
 
-def check_conn(func):
+def _check_conn(func):
     """Used to mock connection errors"""
-
     @functools.wraps(func)
     def func_wrapper(*args, **kwargs):
-        if not args[0]._connected:
+        if not func.__self__.connected:
             raise redis.ConnectionError
```

Comment:
Can you also pass a message with the ConnectionError e.g. "fakeredis is emulating a connection error".

Reply:
Will do. Kinda been thinking that needed a message.